cli/src/print.rs

```diff
@@ -13,7 +13,11 @@ impl<W: Write> Print<W> {
         Print { output }
     }
 
-    pub fn payload(&mut self, payload: Payload) -> Result<()> {
+    pub fn payloads(&mut self, payloads: Vec<Payload>) -> Result<()> {
+        payloads.iter().try_for_each(|p| self.payload(p))
+    }
```

Comment:
```suggestion
    pub fn payloads(&mut self, payloads: &[Payload]) -> Result<()> {
        payloads.iter().try_for_each(|payload| self.payload(payload))
    }
```

good 👍 
Learned `try_for_each`! thank you.

The `Vec<Payload>` variable might be ok to borrow here. Try this `&[Payload]`

Reply:
done. It seems that clippy cannot found this issue when I change it the first time...